### DISCUSSION TITLE
Should `tar_quarto()` allow passing new parameters to `quarto_render()`?

### DISCUSSION THREAD

**Discussion by petrbouchal** [ropensci/tarchetypes#166] [Ideas]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

I noticed that in the upcoming version of the `{quarto}` package, other quarto arguments can be passed to the `quarto` CLI vi a new `quarto_args` argument in the `quarto_render()`. It would be helpful if these were also accessible via `tar_quarto()`. 

A known use case would be passing metadata for parameterised reports, e.g. to adjust the title in reports generated automatically from a data frame/set of values in a branching workflow.

See https://github.com/quarto-dev/quarto-r/blob/fc18963e63190ccb52fe708a2756da5192a7f7d4/NEWS.md?plain=1#L11

**Comment by wlandau** [comment]
Happy to add this to `tar_quarto()` etc. after the next version of `quarto` with support for `quarto_args` is on CRAN.

**Reply by wlandau** [reply]
Implemented as of a30558265c5a62cdf98ea7aed0f7ec421bdc1cd5
